packages/syntax-icons/src/generate-icons/iconTemplate.tsx

```diff
@@ -0,0 +1,20 @@
+import { type Template } from "@svgr/babel-plugin-transform-svg-component";
+
+const iconTemplate: Template = ({ componentName, jsx }, { tpl }) => {
+  return tpl`
+  import { type ComponentProps, forwardRef } from "react";
+  import Icon from "../../../syntax-core/src/Icon/Icon";
+
+  const ${componentName} = forwardRef<
+    SVGSVGElement,
+    Omit<ComponentProps<typeof Icon>, "path">
+  >(({ color, size }, ref) => (
+    <Icon ref={ref} path="${jsx.children[0].openingElement.attributes[0].value.value}" color={color} size={size} />
```

Comment:
@somethiiing this seems a bit dangerous - is there any other way to get the `path`?

Reply:
i forgot to tag you in this part, this was specifically the one where i didnt know how to solve/resolve. wasnt sure if there was a better way to access this

Reply:
this is a kind of ridiculous looking if statement, but it solves the typing error:

![image](https://github.com/Cambly/syntax/assets/15243713/e04220ae-803e-431a-9ce2-0e48344298be)

but because they use AST, (and knowing my knowledge from building the linter), this does feel like the safest/correct-est way to fix it

Reply:
@christianvuerings just leaving this for the night, if you dont respond by the morning, ill merge it as is